lms/templates/static_templates/embargo.html

```diff
@@ -4,5 +4,15 @@
 <%block name="pagetitle">${_("This Course Unavailable In Your Country")}</%block>
 
 <section class="outside-app">
-<p>${_("Our system indicates that you are trying to access an edX course from a country currently subject to U.S. economic and trade sanctions. Unfortunately, at this time edX must comply with export controls, and we cannot allow you to access this particular course. Feel free to browse our catalogue to find other courses you may be interested in taking.")}</p>
+<p>
+${_("Our system indicates that you are trying to access an {platform_name} "
```

Comment:
I could simply take out {platform_name}, so that it reads "Our system indicates that you are trying to access a course from a country currently subject to U.S economic and trade sanctions." Would that work?